litmus-portal/cluster-agents/subscriber/pkg/k8s/operations.go

```diff
@@ -2,42 +2,128 @@ package k8s
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"log"
 	"os"
+	"strings"
+	"sync"
+	"time"
 
 	yaml_converter "github.com/ghodss/yaml"
+	"github.com/sirupsen/logrus"
+	yaml2 "gopkg.in/yaml.v2"
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
+	k8s_errors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/api/meta"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime/serializer/yaml"
+	memory "k8s.io/client-go/discovery/cached"
 	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/restmapper"
-
-	memory "k8s.io/client-go/discovery/cached"
 )
 
 const (
 	ExternAgentConfigName = "agent-config"
+	LiveCheckMaxTries     = 6
```

Comment:
I think we should make the LiveCheckMaxTries configurable by the user

Reply:
We can but this is more of a default value, when the retire period is over it will just crash and restart. So this doesn't have to be configured

Reply:
I mean- user can define how many retries he/she wants from an env

Reply:
I understood that, what I meant is configuring this value doesn't change anything because even after the no. of retries are over the subscriber will restart and again retry. So even if the value is small or large there is no practical change, that's why giving user input is not necessary.